src/LightSensor.h

```diff
@@ -0,0 +1,60 @@
+/**
+ ******************************************************************************
+ * @file    LightSensor.h

```

Comment:
Is it possible to use this sensor as a light sensor ? If it is the case, it can be interesting to add an example.

Reply:
The LightSensor.h file is useless for VL53L0X because it is only a proximity sensor but can be used for VL6180X component that instead is both proximity and light sensor. So in my opinion, LightSensor.h should be removed from the VL53L0X library

Reply:
File removed.